.github/ISSUE_TEMPLATE/2-feature-request.yml

```diff
@@ -0,0 +1,32 @@
+name: 'Feature Request'
+labels: ['triage']
+description: Create a feature request to help Epinio to improve
+title: 'Feature Request: '
+body:
+- type: textarea
+  attributes:
+    label: Is your feature request related to a problem?
```

Comment:
We have something similiar in Epinio, however we don't get many feature requests. It could be useful to have a template for a "backlog" story, with description, acceptance criteria, etc. If we want to commit to those headlines for our planned features.

Reply:
Thanks for the feedback, could you clarify what would be the difference between a "feature request" and a backlog story?